server/src/services/metadata.service.ts

```diff
@@ -584,16 +598,27 @@ export class MetadataService extends BaseService {
     if (timeZone) {
       this.logger.verbose(`Asset ${asset.id} timezone is ${timeZone} (via ${exifTags.tzSource})`);
     } else {
-      this.logger.warn(`Asset ${asset.id} has no time zone information`);
+      this.logger.verbose(`Asset ${asset.id} has no time zone information`);
+    }
+
+    let fileCreatedAt = asset.fileCreatedAt;
+    let fileModifiedAt = asset.fileModifiedAt;
+
+    if (asset.isExternal) {
+      // With external assets we need to extract dates from the filesystem, this can't be done with uploades assets as that information is lost on upload
+      fileCreatedAt = stat.mtime;
+      fileModifiedAt = stat.mtime;
```

Comment:
I would argue that we should still prefer exif dates, no? Those will always be more accurate